### ISSUE TITLE
Link Resource Timing entry

### ISSUE THREAD

**Issue by npm1** [WICG/element-timing#2]
Currently, the Element Timing API exposes only two of the attributes from Resource Timing: `responseEnd` and `name`. For images, it seems useful to instead provide a direct link to the Resource Timing entry corresponding to the element. For text, we will not be able to provide this at first, and in fact text elements may not correspond to any Resource Timing entry. But overall it seems useful to provide a pointer to RT from ET.

This issue is filed based on the discussion at https://github.com/w3c/user-timing/issues/17.

**Comment by sergeychernyshev** [comment]
Text might enjoy a link to a custom font resource used to render it.

**Comment by npm1** [comment]
Correct! I mention that we might not be able to provide this at first because on our text [proposal](https://docs.google.com/document/d/1xhPJnXf0Nqsi8cBFrlzBuHavirOVZBd8TqdD_OyrDGw/edit?usp=sharing) we do not wait until webfonts have loaded. In the future, we could, but text is also a bit trickier because a single block of text (like a paragraph) could be using multiple webfonts (so we'd need an array of RT entries).

**Comment by sergeychernyshev** [comment]
Agreed. Just wanted to weigh in as I see that being important quite often and requires some attention when we use User Timing for this.

Good point about multiple web fonts - not sure how this will handle `font-display: swap` situation though... we currently measure it anyway knowing that there is a difference in experience between browsers that support it and browsers that don't. Same applies to other values, I assume.

**Comment by mmocny** [comment]
This still seems like an interesting request.

There is a related issue that Microsoft engineers brought up related to mapping fetch() requests to resource timings (which is non trivial when you make multiple requests to same url endpoint).

There is a proposal I think to add a resourceTimingEntryId or some such.

**Comment by mmocny** [comment]
@monica-ch FYI, wonder if you could add this to the project considerations
